common/views/themes/config.ts

```diff
@@ -148,7 +148,7 @@ export const themeValues = {
   },
 
   // Keyboard focus uses a hard box shadow of 0.7 opacity 'turquoise'
-  focusBoxShadow: '0 0 0 3px rgba(92, 184, 191, 0.7)',
+  focusBoxShadow: '0 0 0 3px rgba(43, 136, 143, 1)',
```

Comment:
Might be worth running this by @GarethOrmerod (or at least letting him know it's been done). I think he spent some time looking at accessibility considerations for this focus style.

Reply:
I did, [he chose this colour](https://github.com/wellcomecollection/wellcomecollection.org/issues/6741#issuecomment-876575751)